<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <group ns="X1">
  <node pkg="marble_guidance" type="guidance_node.py" name="guidance_controller">
    <param name="vehicle_type" type="string" value="ground" />
    <param name="fixed_frame" type="string" value="world" />
    <param name="speed" value="1.0" />
    <param name="Tstar" value="1.0" />
    <param name="reverse" value="false" />
    <param name="yaw_rate_max" value="2.0" />

    <!-- <remap from="odometry" to="odometry" /> -->
    <!-- <remap from="path" to="ma_goal_path"/> -->
    <!-- <remap from="path" to="planned_path"/> -->
    <remap from="path" to="path_stitched"/>
    <!-- <remap from="goal_pose" to="ma_goal"/> -->
    <!-- <remap from="cmd_vel" to="cmd_vel_guidance"/> -->
    <remap from="lookahead_point" to="lookahead_point"/>
    <remap from="lookahead_vec" to="lookahead_vec"/>
  </node>
  </group>
</launch>
